runtime/libia2/init.c

```diff
@@ -13,7 +15,45 @@
 extern __thread void *ia2_stackptr_0[PAGE_SIZE / sizeof(void *)]
     __attribute__((aligned(4096)));
 
-uintptr_t ia2_stack_addrs[MAX_PKEYS] IA2_SHARED_DATA = {0};
+__thread uintptr_t ia2_stack_addrs[MAX_PKEYS] IA2_SHARED_DATA = {0};
```

Comment:
`IA2_SHARED_DATA` does not work with thread locals at present, see #249.

Reply:
I switched things to avoid this now.